packages/types/src/indexed-db.ts

```diff
@@ -194,7 +195,7 @@ export interface PenumbraDb extends DBSchema {
   };
   GAS_PRICES: {
     key: 'gas_prices';
-    value: GasPrices;
+    value: PlainMessage<GasPrices>;
```

Comment:
Let's make this `Jsonified<GasPrices>`

Reply:
`PlainMessage` should provide better type safety than the de-typed output of `Jsonified` which is simply `JsonObject`